### DISCUSSION TITLE
Intro

### DISCUSSION THREAD

**Discussion by cyberprophet** [Share-Invest/Algorithmic-Trading-Server#5] [Show and tell]
![20230115_013636](https://user-images.githubusercontent.com/48705422/212484450-8da86bfc-0302-4993-856e-8d362f25ba4e.png)

**Comment by mrmind-manager** [comment]
![mobile_dark_mode](https://user-images.githubusercontent.com/45344929/216391133-51b49d4d-bf9e-4b27-9849-86ec0c5bd96b.jpg)
